src/ViewerPlugin.ts

```diff
@@ -0,0 +1,17 @@
+import { App } from "vue";
+import { createPinia } from "pinia";
+import CrucibleComponent from "./components/CrucibleComponent.vue";
+import { ViewerPluginOptions } from "./types";
+
+const defaultDataLink = "http://localhost:8080/api/resource/getQuiz";
```

Comment:
Why is the address hardcoded like this? Isn't this more of an env option?

Reply:
fixed, should not be a api link.